resources/results.py

```diff
@@ -31,20 +34,24 @@
 #     return f"{ccdg_bucket}/vds/{data_type}.vds"
 
 
-def get_ccdg_vds(data_type: str = "genomes") -> hl.vds.VariantDataset:
+def get_ccdg_vds(data_type: str = "genomes", filter : bool = False) -> hl.vds.VariantDataset:
```

Comment:
I like`filter_washu` better (like used in`get_joint_pca_variants_ht_path`), just `filter` is too general